source/_integrations/scene.mqtt.markdown

```diff
@@ -0,0 +1,134 @@
+      type: string
+      default: offline
+    topic:
+      description: An MQTT topic subscribed to receive availability (online/offline) updates.
+      required: true
+      type: string
+availability_topic:
+  description: The MQTT topic subscribed to receive availability (online/offline) updates. Must not be used together with `availability`.
+  required: false
+  type: string
+command_topic:
+  description: The MQTT topic to publish commands to change the switch state.
+  required: false
+  type: string
+icon:
+  description: Icon for the switch.
+  required: false
+  type: icon
+name:
+  description: The name to use when displaying this switch.
+  required: false
+  type: string
+  default: MQTT Switch
+payload_available:
+  description: The payload that represents the available state.
+  required: false
+  type: string
+  default: online
+payload_not_available:
+  description: The payload that represents the unavailable state.
+  required: false
+  type: string
+  default: offline
+payload_on:
+  description: The payload that represents `on` state. If specified, will be used for both comparing to the value in the `state_topic` (see `value_template` and `state_on`  for details) and sending as `on` command to the `command_topic`.
+  required: false
+  type: string
+  default: "ON"
+qos:
+  description: The maximum QoS level of the state topic. Default is 0 and will also be used to publishing messages.
+  required: false
+  type: integer
+  default: 0
+retain:
+  description: If the published message should have the retain flag on or not.
+  required: false
+  type: boolean
+  default: false
+unique_id:
+  description: An ID that uniquely identifies this switch device. If two switches have the same unique ID, Home Assistant will raise an exception.
+  required: false
+  type: string
+{% endconfiguration %}
+
+<div class='note warning'>
+
+Make sure that your topic matches exactly. `some-topic/` and `some-topic` are different topics.
+
+</div>
+
+## Examples
+
+In this section, you will find some real-life examples of how to use this sensor.
+
+### Full configuration
+
+The example below shows a full configuration for a switch.
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: mqtt
+    unique_id: living_room_party_scene
+    name: "Living Room Party Scene"
+    command_topic: "home/living_room/party_scene/set"
+    availability:
+      - topic: "home/living_room/party_scene/available"
+    payload_on: "ON"
+    qos: 0
+    retain: true
+```
+
+### Use with Zigbee2MQTT Scenes
+
+Assuming you have [Zigbee2MQTT Scenes](https://www.zigbee2mqtt.io/information/scenes.html) configured, you can use the following configuration as an example:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+scene:
+  - platform: mqtt
+    name: Living Room Blue Scene
+    unique_id: living_room_blue_scene
+    command_topic: zigbee2mqtt/living_room_group/set
+    payload_on: '{"scene_recall": 1}'
+```
+{% endraw %}
```

Comment:
I think this belongs in the Zigbee2MQTT documentation instead.

Reply:
Thanks for the review! I've removed the Z2M references, and just left a generic example for using JSON payloads instead.